sample.env

```diff
@@ -6,4 +6,6 @@ FIREBASE_CLIENT_EMAIL=INSERT_HERE
 FIREBASE_WEB_API_KEY=INSERT_HERE
 FIREBASE_SERVICE_ACCOUNT_ID=INSERT_HERE
 GOOGLE_API_KEY=your_key
-GEOCODING_KEY=INSERT_HERE
\ No newline at end of file
+GEOCODING_KEY=INSERT_HERE
+S3_ACCESS_KEY_ID=INSERT_HERE
+S3_SECRETE_ACCESS_KEY=INSERT_HERE
```

Comment:
Small typo on SECRET!

Reply:
Ah man my spelling is getting worse and worse, thanks!